src/types/infer.ts

```diff
@@ -497,19 +497,6 @@ export class InferType {
         const a = this.typeOf(node.vLeftExpression);
         const b = this.typeOf(node.vRightExpression);
 
-        if (a instanceof NumericLiteralType && b instanceof NumericLiteralType) {
```

Comment:
I'm worried about moving this line below. We want `InferType.typeOf(`4+3`)  to be `IntLiteral(7)`. I think moving this below would break this

Reply:
I restored original logic in d4a7200 and found workaround in 2fcc3d9.